rollup.config.js

```diff
@@ -20,4 +20,17 @@ export default {
   plugins: [replace({
     TRACING: process.env.sjs ? 'false' : 'true'
   })]
-};
+}, {
```

Comment:
I added a separate config for node-specific settings. The most important part here is that the output format is 'cjs' instead of 'iife'.